server/src/main/resources/db/migration/postgresql/V5_0__Additional_Data_Storage_Info.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE object_permissions
+    ADD COLUMN granter_address VARCHAR(44),
+    ADD COLUMN storage_key_address VARCHAR(44),
+    ADD COLUMN is_object_with_meta BOOLEAN
+;
+
+WITH granter_lookup AS (
```

Comment:
Looks like the passport gateway instance is the only one with object_permissions records, and all those were granted by the one registered key when doing some queries over there

Reply:
and I ran the migration against that data locally, and it seemed to work fine